README.md

```diff
@@ -1,4 +1,4 @@
-# Mochi
+# Mochii
```

Comment:
i think we should be consistent lol and call it one thing.

Below i document a `mochi` function that has my defaults.